vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/ConsumableResource.java

```diff
@@ -78,7 +78,7 @@ public JavaType typeFromId(DatabindContext context, String id) throws IOExceptio
    * @param workflowVersion the version of the workflow
    * @param vidarrId the identifier of the workflow run
    */
-  void release(String workflowName, String workflowVersion, String vidarrId);
+  void release(String workflowName, String workflowVersion, String vidarrId, Optional<JsonNode> resourceJson);
```

Comment:
how is resourceJson different from getting inputFromSubmitter on line 62?

Reply:
resourceJson would be the actual value of the resource, whereas inputFromSubmitter returns the type not the value itself

Reply:
Why does release need to know what the value was?

Reply:
Because it is adding it to the wait list (sorted set) when it is released - so it needs to store it's priority in that wait list